### PR TITLE
Fix compilation on visual c++

### PR DESCRIPTION
**Pull request by tjachmann**

### CODE REVIEW

File: naett.c [hunk 1]
```diff
@@ -34,6 +34,12 @@
 
 #define naettAlloc(TYPE, VAR) TYPE* VAR = (TYPE*)calloc(1, sizeof(TYPE))
 
+#ifdef _MSC_VER 
+    #define strncasecmp _strnicmp
```
Comment (erkkah) [inline naett.c:38]: This one is not used, right?

File: naett.c [hunk 2]
```diff
@@ -34,6 +34,12 @@
 
 #define naettAlloc(TYPE, VAR) TYPE* VAR = (TYPE*)calloc(1, sizeof(TYPE))
 
+#ifdef _MSC_VER
```
Comment (erkkah) [inline naett.c:37]: Can you move this into the `#ifdef _WIN32` section above?

**Review by erkkah — CHANGES_REQUESTED** [review]
Hi. Thanks for the PR, and for making it build on VC++. I've not had a chance to test it myself, I'm glad it did not require that many changes.

File: naett.c (hunk 2 above)
Comment (tjachmann) [reply naett.c:37]: That would be slightly less correct. The fixes are for the microsoft compiler/its headers. They are not windows specific. Although makes not much difference as of now...

File: naett.c (hunk 1 above)
Comment (tjachmann) [reply naett.c:38]: A yes that can be dropped.

File: naett.c (hunk 2 above)
Comment (erkkah) [reply naett.c]: Fair enough :)

**Review by erkkah — APPROVED** [review]

**Comment by erkkah** [comment]
Thanks again, @tjachmann !

**Pull request merged**